techmatters/webchat
# Web page language detection

## Description

new feature: Web page language detection

### Checklist
- [N/A] New tests added



### Related Issues
Fixes CHI-1070

### Verification steps


In the `index.html` file, add the `data-language` attribute within the script tag, and set a language as the value. E.g., `data-language="ukr-HU"` save and verify that the web chat is set to the correct translation.

The [Aselo Webchat Integration](https://app.box.com/notes/850639952620?s=y9qf67jmj5pfqhczgd72g58yt3uf3qxt) document was updated to reflect the changes.
